packages/core/utils/src/convert-query-params.ts

```diff
@@ -354,7 +358,7 @@ const createTransformer = ({ getModel }: TransformerOptions) => {
         try {
           const subPopulateAsBoolean = parseType({ type: 'boolean', value: subPopulate });
           // Only true is accepted as a boolean populate value
-          return subPopulateAsBoolean === true ? { ...acc, [key]: true } : acc;
+          return subPopulateAsBoolean ? { ...acc, [key]: true } : acc;
```

Comment:
Is this necessary? Shouldn't parseType() always return a boolean? And if it's '*' won't it evaluate to false (assuming that's the reason for the change)

Reply:
The reason for the change was to make lints happy

At this point it should always be a boolean (as it would throw if the string wasn't a valid bool representation, thus not reaching this point)